TabloidCLI/Repositories/PostRepository.cs

```diff
@@ -2,16 +2,51 @@
 using System.Collections.Generic;
 using Microsoft.Data.SqlClient;
 using TabloidCLI.Models;
+using TabloidCLI.Repositories;
 
-namespace TabloidCLI.Repositories
+
+namespace TabloidCLI
```

Comment:
Curious why the change on the namespace here?

Reply:
I noticed this was done in the author repository, so I did it in the post repository for consistency. But it could be updated back to what it initially was, and I can refactor.

Reply:
Seems like an oversight... if it's a repository, it makes sense that all of the repositories would be in the Repositories namespace.

Reply:
Just reviewed, and it looks like the pre-populated TagsRepository has the same format  as the prepopulated AuthorsRepository. I'll update the Posts to be within the Repositories namespace.

Reply:
yeah, there are a number of the repos that seem to be like that.